platform/linuxbsd/x11/detect_prime_x11.cpp

```diff
@@ -208,7 +208,10 @@ int detect_prime() {
 				print_verbose("Couldn't write vendor/renderer string.");
 			}
 			close(fdset[1]);
-			exit(0);
+
+			//quick_exit() is used because exit() will call destructors on static objects copied by fork()
```

Comment:
Please amend the comment to use the same style as surrounding code (capitalized sentences ending with a period, space after `//`).